### DISCUSSION TITLE
Compatibility Testing!

### DISCUSSION THREAD

**Discussion by FxMorin** [FxMorin/MoreCulling#31] [General]
This issue will be used as a list of mods that currently don't support MoreCulling so that I can go through the mods and fix the issues.

### Supported Rendering Engines
- Vanilla
- [Fabric-Api](https://github.com/FabricMC/fabric/tree/1.19/fabric-renderer-indigo) (Indigo)
- [Sodium](https://github.com/CaffeineMC/sodium-fabric) (sodium) Added in v0.1.4
- [Indium](https://github.com/comp500/Indium)

### Currently Unsupported Mods
- MultiPartLib (multi-parts will not culled, still works fine)

### Currently Unsupported Rendering Engines
- None

### Rendering Engines that need testing
- [FREX](https://github.com/vram-guild/frex) (pastal)
- [Canvas](https://github.com/vram-guild/canvas) (pastal + canvas)
- [VulkanMod](https://github.com/xCollateral/VulkanMod)
- [Iris](https://github.com/IrisShaders/Iris)
- [Blaze4D](https://github.com/KilnGraphics/Blaze4D)

### Mods that need testing
- [BlocksToParts](https://github.com/Juuxel/BlocksToParts)
- [connected-block-textures](https://github.com/TwilightFlower/connected-block-textures)

---

If you want to test if the mod is working, an easy way to do this is by using a chorus plant, and testing [this bug](https://bugs.mojang.com/browse/MC-249693)
There might be an automated system in the future, although don't get your hopes up

**Comment by rayzchen** [comment]
Dashloader is incompatible, it cannot load this mod's cache for some reason, the game still works just dashloader breaks

**Reply by FxMorin** [reply]
has been fixed

**Comment by JL2210** [comment]
Just FYI, this mod loads and runs with VulkanMod enabled on 1.20.1. Chorus fruit test also appears to function. Can't vouch for the performance impact.

**Comment by AlexanderwithaA** [comment]
The only thing I've found broken with vulkanmod is that when you try to access moreculling's options through modmenu the game crashes. Here's a log if you want to take a look. https://mclo.gs/1HVdkAT

**Comment by Margeshy** [comment]
This is incompatible with Forge through the usage of Sinytra's Connector
